.travis.php

```diff
@@ -21,6 +21,10 @@
 
     $json = file_get_contents($dir.'/composer.json');
     $package = json_decode($json);
+    if (!$package || '{' !== $json[0]) {
+        echo "Invalid composer.json\n";
```

Comment:
I don't agree. Why shouldn't we make it easier to spot the error? I mean why should someone has to use another linter just to recognise what is actually wrong with the `composer.json` file?